content/purgatorio/now-go-build/a-messy-real-world.md

```diff
@@ -1,5 +1,185 @@
+On the first day of work, your boss gives you a "dataset" that a client gave him and asks you to get "**an AI who understands how to make them useful**" (what?!?).
+
+The dataset has a lot of missing values, the column names are flimsy and nobody explained how to interpret them. Exploring those messy data you find that the column to predict has half the missing values, and the other columns have enough noise to make them unusable.
+
+So this dataset, with your actual knowledge, is almost unusable!
+
+Moreover, **the salesmen were not aligned on the current possibilities of Artificial Intelligence** (as they call predictive models to sell them at higher price :D), and they promised the moon!
+
+---
+
+Sad and desolate you take the negative comments of your boss, who thinks that you don't know how to do anything.
+
+
+What went wrong? Wasn't everything as rosy as you thought?
+
+It's everything fine, you just need some **reality check.** 
+
+
+::: tip
+If you've ever had a situation like this, well, it's probably better if you move away from such a boss). 
+:::
+
+## Reality Check
+
+For sure you've heard that [Data Scientist is The Sexiest Job of the 21st Century](https://hbr.org/2012/10/data-scientist-the-sexiest-job-of-the-21st-century), and a lot of articles and blogs describes the Data Scientist daily job as a walk in the park, without messiness and effort.
+
+**The inconvenient truth is completely different, and here are some reasons:**
+
+- Most of the Data Scientist's daily time [is related to ETL](https://www.quora.com/What-is-an-average-work-day-for-a-data-scientist) (Extract, Transform, and Load Data), not the funniest thing ever.
+- Real-world datasets are rarely complete, clean, well aggregated ([Machine learning isn't Kaggle competitions](http://jvns.ca/blog/2014/06/19/machine-learning-isnt-kaggle-competitions/)).
+- You don't have someone explaining how they were built or what to expect inside them (like a Kaggle "data explanation" section). 
+- Often the datasets you are provided contain a lot of noise, and it is vital to know what to discard. 
+- Everyone is [expecting you to do fancy magic with data](https://www.reddit.com/r/datascience/comments/3m78k1/i_hire_data_scientists_this_is_the_stuff_this/), maybe they expect you to build a super-intelligent AI to fire employees and take over the world.
+
+Okay, maybe the last one's a joke... or not! 
+
+Since there is a huge hype around the field of Data Science, **the expectations** of those who allocate the company budget to this topic are also very high!
+
+See:
+
+- [Here’s why so many data scientists are leaving their jobs](https://towardsdatascience.com/why-so-many-data-scientists-are-leaving-their-jobs-a1f0329d7ea4)
+
+::: warning 
+That **doesn't mean** you have to be discouraged, it just means you have to have the right expectations! With the right expectations and premises, the work of Data Scientist is _challenging, satisfying and creates enormous value for those who use it!_
```

Comment:
👍 
right expectations and premises ==> check if this is the right construction, "premise" might be the right word here

Reply:
Grammarly says that's ok :D